Polished intro to SDKs (#966)
casper-network/docs: source/docs/casper/developers/dapps/sdk/index.md

---
title: Introduction to SDKs
slug: /sdk
---

> # Introduction to SDKs

Reviewer: The title didn't revert for some reason, so I will do it.
```suggestion
# SDK Client Libraries
```

---

> title: Introduction to SDKs

Reviewer: What is the reasoning behind changing the title? I think it would be best to keep the original one.
This is not really an introduction to what SDKs are... but just listing the available libraries.

Reviewer 2: My mistake, I thought I'd done this. The title is now reverted.

title: Introduction to SDKs
slug: /sdk
---

# SDK Client Libraries
# Introduction to SDKs

This section covers the software development kits (SDKs) published by third parties available for interacting with the Casper blockchain. These SDKs are client-side libraries providing functions or methods (depending on the language) to interact with a Casper network. You can use them as a model to develop your application and accomplish tasks such as generating account keys, sending transfers, or other blockchain transactions.

> Each such third party is solely responsible for the SDK it provides, any warranties (to the extent that such warranties have not been disclaimed), and for any claims you may have relating to your access or use thereof. We do not approve or endorse any such SDKs by providing a link thereto, and assume no responsibility for your access or use thereof. The SDKs may be subject to additional licenses and disclaimers as set out in the relevant GitHub repositories.

Reviewer: Please bring this disclaimer back up, it's important for readers to see it before they jump into the links.

> Each of these SDKs can be used to build dApps. For browser interaction you can use the JavaScript SDK, for desktop applications there is C#, Java, etc. Click the link on your preferred SDK to learn how to use it in dApp development. You may visit the SDKs' Github repositories to delve into the source code.

Reviewer: ```suggestion
Each of these SDKs can be used to build dApps. For browser interaction, you can use the JavaScript SDK; for desktop applications, there are C# and Java SDKs. Click the link on your preferred SDK to learn how to use it in dApp development. To delve into the source code, you may visit the SDKs' Github repositories.
```

| SDK Documentation      | GitHub Location      | Organization |
| ---------------------- | -------------------- | ---------- |
|[JavaScript/TypeScript](./script-sdk.md) | [casper-js-sdk](https://github.com/casper-ecosystem/casper-js-sdk/)| [Casper Ecosystem](https://github.com/casper-ecosystem) |
|Java SDK | [casper-java-sdk](https://github.com/casper-network/casper-java-sdk/)| [Casper Association](https://github.com/casper-network)|
|[C# SDK](./csharp-sdk.md)|[casper-net-sdk](https://github.com/make-software/casper-net-sdk)| [MAKE](https://github.com/make-software) |
|[Golang SDK](./go-sdk.md) |[casper-golang-sdk](https://github.com/casper-ecosystem/casper-golang-sdk/)| [Casper Ecosystem](https://github.com/casper-ecosystem) |
|[Python SDK](./python-sdk.md) |[casper-python-sdk](https://github.com/casper-network/casper-python-sdk/)| [Casper Association](https://github.com/casper-network) |
|PHP SDK|[casper-php-sdk](https://github.com/make-software/casper-php-sdk)| [MAKE](https://github.com/make-software) |

> | Scala SDK | [casper-scala-sdk](https://github.com/abahmanem/casper-scala-sdk) | [M. Abahmane](https://github.com/abahmanem) |

Reviewer: Please revert changes to this table. It seems to be old content that wasn't merged correctly. See PR [896](https://github.com/casper-network/docs/pull/896).
